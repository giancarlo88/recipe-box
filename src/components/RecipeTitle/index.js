import React from 'react'
import styled from 'styled-components'

const StyledRecipeTitle = styled.h2`
  font-weight: bold;
  font-size: 1.5em;
  padding: 15px 0;
`

const RecipeTitle = ({ text }) => <StyledRecipeTitle>{text}</StyledRecipeTitle>

export default RecipeTitle
