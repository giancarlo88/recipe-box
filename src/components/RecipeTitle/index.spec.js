import React from 'react'
import { create } from 'react-test-renderer'
import { mount } from 'enzyme'

import RecipeTitle from '.'

describe('<RecipeTitle />', () => {
  const text = 'My recipe'
  it('should match a snapshot', () => {
    const props = {
      text
    }
    const tree = create(<RecipeTitle {...props} />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the text it receives as a prop', () => {
    const component = mount(<RecipeTitle text={text} />)
    expect(component.find('h2').text()).toBe(text)
  })
})
