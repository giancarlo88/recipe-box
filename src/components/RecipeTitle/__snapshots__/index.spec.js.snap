// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecipeTitle /> should match a snapshot 1`] = `
<h2
  className="sc-bdVaJa fEgdCy"
>
  My recipe
</h2>
`;
